Gemfile

```diff
@@ -1,27 +1,29 @@
 source 'https://rubygems.org'
 
-ruby "2.7.8"
+ruby "3.3.8"
 
-gem 'rails', '~> 6.1.0.0'
+gem "rails", "~> 6.1.7"
```

Comment:
Upgraded to latest in 6.1.x instead of just 6.1.0. Smooths some ruby3+ issues too.